:sparkles: Add endpoints for c4p2n triggers (#4)
piterpy-meetup/ppm-telegram-bot: README.md

# Welcome to ppm-telegram-bot 👋
![License: MIT](https://img.shields.io/badge/License-MIT-yellow.svg)

> > Integration with telegram via bot platform. Uses for internal meetup needs

Reviewer: ```suggestion
> Telegram Bot Platform integration for bot commands processing. We use it for our internal @piterpy-meetup needs, basically, to manage things via Telegram.
```

## Install

> We use `poetry` as dependency manager, so it's the fastest way to install all stuff:

Reviewer: ```suggestion
We use `poetry` to manage dependencies, so it's the fastest way to install them:
```

```sh
poetry install
```

## Usage
In general, app is just FastAPI-based web-app with aiogram for interaction with telegram. So, for running it, you can

>  use any ASGI server, e.g. uvicorn:

Reviewer: ```suggestion
In general, ppm-telegram-bot is just a FastAPI web app with aiogram to interact with Telegram.

To run it you can use any ASGI server like [uvicorn](https://www.uvicorn.org/),
[gunicorn with uvicorn workers](https://www.uvicorn.org/deployment/#gunicorn)
or [daphne](https://github.com/django/daphne/#running), e.g.:
```

Author: Why? I add uvicorn just as an example of ASGI server and don't want to list all ASGI servers 🤔

Reviewer: Well, actually, yeah, it might not be a good idea to _list_ all of them. But then, I'd rather cover the gunicorn with uvicorn running, instead, as a more production-suitable solution.

Author: LGTM

```sh
uvicorn ppm_telegram_bit.api:app
```

## Deployment

> We currently use awesome [kintohub](https://kintohub.com) to run a docker container as serverless web-app. 

Reviewer: ```suggestion
We currently use awesome [kintohub](https://kintohub.com) to run the app in a Docker container as a serverless web app. 
```

## Development

> Please, carefully read docstring in all modules: they usually contains info and hints.

Reviewer: ```suggestion
Please, carefully read docstrings in all modules, they usually contain useful info and hints.
```

## Show your support

> Give a ⭐️ if you like this project

Reviewer: ```suggestion
Give us a ⭐️ if you like this project!
```
